app/Jobs/Ninja/BankTransactionSync.php

```diff
@@ -47,17 +49,29 @@ public function handle()
         foreach (MultiDB::$dbs as $db) {
             MultiDB::setDB($db);
 
-            nlog("syncing transactions");
+            if (Ninja::isSelfHost()) { // @turbo124 @todo I migrated the schedule for the job within the kernel to execute on all platforms and use the same expression here to determine if yodlee can run or not. Please chek/verify
```

Comment:
@turbo124 @todo I migrated the schedule for the job within the kernel to execute on all platforms and use the same expression here to determine if yodlee can run or not. Please chek/verify

Reply:
This looks fine